app/models/persistent_volume_claim.rb

```diff
@@ -0,0 +1,6 @@
+class PersistentVolumeClaim < ActiveRecord::Base
```

Comment:
or even `ContainerVolumePersistentClaim` in order to have it closer to `ContainerVolume` 
the best choice would be to namespace all that container stuff, but this is too far down the road...